SocialCareCaseViewerApi.Tests/V1/Boundary/Request/UpdateMashReferralTests.cs

```diff
@@ -96,7 +96,62 @@ public void WhenUpdateTypeIsInitialDecisionReferralCategoryMustNotBeNull()
         [Test]
         public void WhenUpdateTypeIsInitialDecisionReferralCategoryMustHaveALength()
         {
-            var request = TestHelpers.CreateUpdateMashReferral(updateType: "INITIAL-DECISION");
+            var request = TestHelpers.CreateUpdateMashReferral(updateType: "FINAL-DECISION");
+            request.ReferralCategory = "";
+
+            var validationResult = _validator.Validate(request);
+
+            validationResult.ToString().Should().Be("Must provide a referral category");
+        }
+
+        [Test]
+        public void WhenUpdateTypeIsInitialDecisionADecisionMustNotBeNull()
+        {
+            var request = TestHelpers.CreateUpdateMashReferral(updateType: "FINAL-DECISION");
+            request.Decision = null;
+
+            var validationResult = _validator.Validate(request);
+
+            validationResult.ToString().Should().Be("Must provide a decision");
+        }
+
+        [Test]
+        public void WhenUpdateTypeIsInitialDecisionDecisionMustHaveALength()
+        {
+            var request = TestHelpers.CreateUpdateMashReferral(updateType: "FINAL-DECISION");
+            request.Decision = "";
+
+            var validationResult = _validator.Validate(request);
+
+            validationResult.ToString().Should().Be("Must provide a decision");
+        }
+
+        [Test]
+        public void WhenUpdateTypeIsFinalDecisionRequiresUrgentContactMustBeProvided()
+        {
+            var request = TestHelpers.CreateUpdateMashReferral(updateType: "FINAL-DECISION");
+            request.RequiresUrgentContact = null;
+
+            var validationResult = _validator.Validate(request);
+
+            validationResult.ToString().Should().Be("Must provide if urgent contact is required");
+        }
+
+        [Test]
+        public void WhenUpdateTypeIsFinalDecisionReferralCategoryMustNotBeNull()
```

Comment:
This looks like it shares the same setup as the test on line 86. Not sure if we've duplicated the test (it has a different test name but same setup as this)or maybe we've passed in the wrong updateType for that test on line 86?